api-mock/mocks/AccountLinkingMock.ts

```diff
@@ -0,0 +1,140 @@
+            input: {
+                recipeUserId: recipeUserId.getAsString(),
+                primaryUserId,
+                userContext,
+            },
+        });
+        return {
+            ...response,
+            user: response.user && new UserClass(response.user),
+        };
+    },
+    isEmailChangeAllowed: async (recipeUserId, newEmail, isVerified, session, userContext) => {
+        return await queryAPI({
+            method: "post",
+            path: "/mock/accountlinking/isemailchangeallowed",
+            input: {
+                recipeUserId: recipeUserId.getAsString(),
+                newEmail,
+                isVerified,
+                session,
+                userContext,
+            },
+        });
+    },
+    unlinkAccount: async (recipeUserId, userContext) => {
+        return await queryAPI({
+            method: "post",
+            path: "/mock/accountlinking/unlinkaccount",
+            input: {
+                recipeUserId: recipeUserId.getAsString(),
+                userContext,
+            },
+        });
+    },
+    createPrimaryUserIdOrLinkAccounts: async (tenantId, recipeUserId, session, userContext) => {
+        return await queryAPI({
+            method: "post",
+            path: "/mock/accountlinking/createprimaryuseridorlinkaccounts",
+            input: { tenantId, recipeUserId: recipeUserId.getAsString(), session, userContext },
+        });
+    },
+    getPrimaryUserThatCanBeLinkedToRecipeUserId: async (tenantId, recipeUserId, userContext) => {
+        const user = await queryAPI({
+            method: "post",
+            path: "/mock/accountlinking/getprimaryuserthatcanbelinkedtorecipeuserid",
+            input: {
+                tenantId,
+                recipeUserId: recipeUserId.getAsString(),
+                userContext,
+            },
+        });
+        return user === undefined ? undefined : new UserClass(user);
+    },
+    isSignUpAllowed: async (tenantId, newUser, isVerified, session, userContext) => {
+        return await queryAPI({
+            method: "post",
+            path: "/mock/accountlinking/issignupallowed",
+            input: { tenantId, newUser, isVerified, session, userContext },
+        });
+    },
+    isSignInAllowed: async (tenantId, recipeUserId, session, userContext) => {
+        return await queryAPI({
+            method: "post",
+            path: "/mock/accountlinking/issigninallowed",
+            input: { tenantId, recipeUserId: recipeUserId.getAsString(), session, userContext },
+        });
+    },
+    canCreatePrimaryUser: async (recipeUserId, userContext) => {
+        return await queryAPI({
+            method: "post",
+            path: "/mock/accountlinking/cancreateprimaryuser",
+            input: {
+                recipeUserId: recipeUserId.getAsString(),
+                userContext,
+            },
+        });
+    },
+};
+
+export const AccountLinkingRecipeMock = {
+    // @ts-ignore
+    getInstance: () =>
```

Comment:
Ideally we should drop `getInstance` in these cases

Reply:
why? I mean, the tests are using the AccountLinkinigRecipe.getInstance().whatever 
I don't want to change that much because is going to be hard for doing tests since we're no going to be able to use same functions as you do with the sdk

Reply:
I didn't create the tests, my goal was to change as minimal as possible. We may advise all developers to avoid using getInstance on the tests